source/draw/RasterYuvSemiplanar.ooc

```diff
@@ -50,7 +50,9 @@ RasterYuvSemiplanar: abstract class extends RasterPlanar {
 		this uv referenceCount decrease()
 		super()
 	}
-
+	apply: override func ~rgb (action: Func (ColorRgb)) {
+		this apply(ColorConvert fromYuv(action))
```

Comment:
I know, but when it comes to memory management in ooc I always get confused, especially when it's related to closures. I always need valgrind to confirm.